flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/utils/JarHandlerUtils.java

```diff
@@ -147,16 +149,14 @@ public void applyToConfiguration(
             configuration.set(CoreOptions.DEFAULT_PARALLELISM, parallelism);
 
             final PackagedProgram program = toPackagedProgram(configuration);
+            final List<URL> classpaths = getClasspaths(configuration);
```

Comment:
I guess the alternative would be to fix the `toPackagedProgram` logic to set the class paths.

Reply:
Maybe that would be better to have a cleaner logic